### PR TITLE
v1.0.3

### PR DESCRIPTION
**Pull request by Portals**
Thanks for helping with creating `v1.0.3` @molleer , @Sponken, and @lir108!

Resolves a lot of issues with Gamma helpers. They were basically unusable in `v1.0.2` unfortunately. But here are the relevant issues that were resolved that affects you the user:
- #350 Adds documentation about size, flex, margin, padding, justifySelf and alignSelf.
- #393 Add `<DigitRating />`
- #400 Add documentation about extraButton in `<DigitEditDataCard />`
- #408 Add a `signOut` property for `<DigitGammaActions />`
- #409 Resolve bug with loading wasn't set to false when `redirectAutomatically` was set to true.
- #410 Add `gridRow` and `gridColumn` to each component so that they easily can integrate with `display: grid;`.
- #415 Add missing proptypes for `<DigitButton />`
- #419 Resolve issue with search of `<DigitAutoCompleteSingle />`/`<DigitAutocompleteMultiple />`. 
- #421 Resolve issue with `<DigitTextField />` / `<DigitTextArea />` where a null value and a max length would crash
- #423 Resolve issue where one couldn't use `<DigitDateAndTimePicker />` inside a dialog.
- #425 Resolve issue with pressing a details button in a `<DigitSelectMultipleTable />` would select the checkbox
- #426 Add missing documentation for `<DigitGammaActions />`

- Add `<DigitSwipeableDrawer />`
- Add `disableReadOne` to `<DigitCRUD />`
- Add `overflow` prop to `<DigitDesign.Card />` and `<DigitLayout.Flex />`
- Add `cardProps` and `cardBodyProps` where you can edit the `<DigitDesign.Card />` and `<DigitDesign.CardBody />` respectively inside `<DigitEditDataCard />`
- Add the possibility to use `formatEvent` for `<DigitEditData />`/`<DigitEditDataCard />`.

### CODE REVIEW

**Comment by Portals** [comment]
Please wait to merge until `v1.0.3` commit has been created.

File: src/declaratives/digit-providers/DigitProviders.declarative.jsx [hunk 1]
```diff
@@ -79,9 +79,9 @@ const DigitProviders = ({
                                                 {children}
                                             </BrowserRouter>
                                         )}
```
Comment (ericlp) [inline src/declaratives/digit-providers/DigitProviders.declarative.jsx:81]: I had a problem where I couldn't use `Link` in DigitDialog. I assume this problem originates from `<DigitDialogContextSingletonProvider>` being placed outside the router. Is there some reason why `<DigitDialogContextSingletonProvider>`  this structure is in place or could the provider be moved inside the router?

Comment (Portals) [reply src/declaratives/digit-providers/DigitProviders.declarative.jsx:81]: Nope. I will move up the router.

**Comment by Portals** [comment]
Published  `v1.0.3-rc.1`. @ericlp it resolves the router issue you had.

**Review by ericlp — APPROVED** [review]

**Comment by Portals** [comment]
@molleer @Hoidi ?

**Pull request merged**